CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java

```diff
@@ -761,7 +761,8 @@ private void partition(final int recursiveCount, final boolean fast, final boole
         if (verbose) {
             if (config.getVerbosity() == 0) {
                 final String fmt = String.format("%s.%df", "%", config.getVerboseNumberPrecision());
-                System.out.printf("to %d modules with codelength " + fmt + "%n", getNumTopModules(), codelength);
+                final String cl = String.format(fmt, codelength);
+                System.out.printf("to %d modules with codelength %s%n", getNumTopModules(), cl);
```

Comment:
Hopefully this one is alright. I don't think it's the cleanest solution but I'm struggling to come up with a better one right now

Reply:
This was what I was thinking. Let me know if I've missed the mark:

`System.out.printf("to %d modules with codelength %s.%df%n", getNumTopModules(), "%", config.getVerboseNumberPrecision());`

Reply:
The issue is that it leaves out the codelength variable. %s.%df is only used to get the correct formatter for codelength and therefore I believe there needs to an additional step to apply codelength to the given format